standalone/WebSocketExportToOrg.py

```diff
@@ -0,0 +1,142 @@
+      newSummary.append("\t - URI: " + handshakeMessage.getRequestHeader().getMethod() + " " + handshakeMessage.getRequestHeader().getURI().toString() + "\n")
+      newSummary.append("\t - Port: " + str(handshakeMessage.getRequestHeader().getURI().getPort()) + "\n")
+      newSummary.append("\t - Websocket Port: " + str(channel.port) +"\n")
+      newSummary.append("\t - Websocket Url: " + str(channel.url) + "\n")
+      newSummary.append("\t - Messages Opcodes: \n")
+      newSummary.append("\t - Outgoing Messages format: \n")
+      newSummary.append("\t - Incoming Messages format: \n")
+      newSummary.append("\t - Input Messages encryption: \n")
+      newSummary.append("\t - Output Messages encryption: \n")
+      newSummary.append("\t - Ping/Pong: \n")
+      newSummary.append("\t - Close message: Opcode= , Text= \n")
+      newChannelElement.append_clean(newSummary);
+    
+    # Print handshake request
+    newHandShake = PyOrgMode.OrgNode.Element()
+    newHandShake.heading = "Switching Protocol"
+    newHandShake.level = 2
+
+    # Print Request Header 
+    newHandShakeRequest = PyOrgMode.OrgNode.Element()
+    newHandShakeRequest.heading = "Handshake Request"
+    newHandShakeRequest.append("\t#+BEGIN_SRC html \n" + handshakeMessage.getRequestHeader().toString() + "\t#+END_SRC \n") 
+    newHandShake.append_clean(newHandShakeRequest)
+
+    # Print Response Header 
+    newHandShakeResponse = PyOrgMode.OrgNode.Element()
+    newHandShakeResponse.heading = "Handshake Response"
+    newHandShakeResponse.append("\t#+BEGIN_SRC html \n" + handshakeMessage.getResponseHeader().toString() + "\t#+END_SRC \n") 
+    newHandShake.append_clean(newHandShakeResponse)
+
+    newChannelElement.append_clean(newHandShake)
+    
+  print("Processing Channel ID: #" + str(channel.id))
+  
+  #Get Messages  
+  messageCriteria = WebSocketMessageDTO(channel)
+
+  #messagesList = extWebSocket.getWebsocketMessages(messageCriteria,None, None,None,0,0,10000) #Take a look at https://github.com/zaproxy/zap-extensions/pull/15065
+  messagesList = extWebSocket.getWebsocketMessages(messageCriteria, None,None,0,0,10000)
+
+  newMessages = PyOrgMode.OrgNode.Element()
+  newMessages.level = 2
+  newMessages.heading = "Messages"
+
+  for message in messagesList:
+    if(message.isOutgoing):
+      direction = "OUTGOING"
+    else:
+      direction = "INCOMING"
+
+    newMessage = PyOrgMode.OrgNode.Element()
+    newMessage.level = 3
+    newMessage.heading = "(#" + str(channel.id) + "." + str(message.id) +") "+ direction + " " + message.readableOpcode +" "+ message.dateTime +" "+ str(message.payloadLength) 
+
+    #messageUTF8 = message.getPayloadAsString() #Take a look at https://github.com/zaproxy/zap-extensions/pull/1637
+    try:
+      messageUTF8 = messages.getReadablePayload()
+    except:
+      messageUTF8 = "<unreadable binary payload>"
+    messageHex = message.payload
```

Comment:
Just noticed that this is not being used, but should when writing the message in hex?

Reply:
Do you mean that will be better to use messageHex instead of binascii.hexlify(messageUTF8) ?

Reply:
It's preferable to use `messageHex` instead of `messageUTF8` (the latter might not have the actual contents), independently how it's saved later.